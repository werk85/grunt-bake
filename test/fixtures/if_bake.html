<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<!--(bake includes/include-one.html _if="title")-->
		<!--(bake includes/include-two.html _if="foo")-->
		<!--(bake includes/include-two.html _if="bar")-->

		<!--(bake-start _if="title")-->
		<!-- Here is a comment -->
		<div>Here is some content</div>
		<!--(bake-end)-->

		<!--(bake includes/include-two.html _if="!bar")-->

		<!--(bake-start _if="title == 'Apple Pie'")-->
		<h1>{{title}}</h1>
		<!--(bake-end)-->

		<!--(bake includes/include-two.html _if="title != 'Orange Pie'")-->
		<!--(bake includes/include-one.html _if="title == 'Orange Pie'")-->

		<!--(bake includes/include-one.html _if="'Apple Pie' == title && bar")-->
		<!--(bake includes/include-two.html _if="'Apple Pie' == title && !bar")-->

		Index 0: <!--(bake-start _if="0%2 === 0")-->check<!--(bake-end)-->
		Index 1: <!--(bake-start _if="1%2 === 0")-->check<!--(bake-end)-->
		Index 2: <!--(bake-start _if="2%2 === 0")-->check<!--(bake-end)-->
		Index 3: <!--(bake-start _if="3%2 === 0")-->check<!--(bake-end)-->
		Index 4: <!--(bake-start _if="4%2 === 0")-->check<!--(bake-end)-->
	</body>
</html>