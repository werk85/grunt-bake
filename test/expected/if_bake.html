<!DOCTYPE html>
<html>
	<head></head>
	<body>
		Include One

		<!-- Here is a comment -->
		<div>Here is some content</div>

		Include Two

		<h1>Apple Pie</h1>

		Include Two

		Include Two

		Index 0:check
		Index 1:
		Index 2:check
		Index 3:
		Index 4:check
	</body>
</html>